js/testing_1_2_3.js

```diff
@@ -0,0 +1,14 @@
+// https://www.codewars.com/kata/testing-1-2-3/train/javascript
+
+var number=function(array){
+  let i = 1;
+  const newArray = array.map(function(el){
```

Comment:
Is it necessary to create a `newArray` variable? How about using `return array.map(...` directly?